test/functional/verify/controls/azure_network_security_group.rb

```diff
@@ -0,0 +1,24 @@
+resource_group = attribute('resource_group',         default: nil)
+nsg            = attribute('network_security_group', default: nil)
```

Comment:
Are we aligning the `default:nil`?

Reply:
I think so. Is there a spot where it's missing?

Reply:
Ahh I see. Fixed.